client/my-sites/sidebar/index.jsx

```diff
@@ -565,13 +562,13 @@ export class MySitesSidebar extends Component {
 		}
 
 		// Hide the plan name only for Jetpack sites that are not Atomic or VIP.
-		const displayPlanName = ! ( isJetpack && ! isAtomicSite && ! isVip );
+		const displayPlanName = ! isJetpack || isAtomicSite || isVip;
```

Comment:
This change wasn't strictly necessary, but I did it to reduce some of the overhead of figuring out what this statement means.